.github/dependabot.yml

```diff
@@ -42,6 +44,53 @@ updates:
         exclude-patterns:
           - "@playwright*"
           - "playwright*"
+    target-branch: main
+  # maintenance branch - 4.0.x
```

Comment:
will this comment go obsolete sooner or later? (below the same.

Reply:
at the time the comment will become obsolete, the configuration will become obsolete too